UnitTests/Views/ViewDisposalTest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Reflection;
+using Terminal.Gui;
+using Xunit;
+using Console = Terminal.Gui.FakeConsole;
+using Xunit.Sdk;
+using Microsoft.VisualStudio.TestPlatform.Utilities;
+using Xunit.Abstractions;
+using System.Threading;
+using Terminal.Gui.Graphs;
+
+namespace UnitTests.ViewsTests {
+	public class ViewDisposalTest {
+		ITestOutputHelper output;
+		public ViewDisposalTest (ITestOutputHelper output)
+		{
+			{
+				this.output = output;
+			}
+		}
+#nullable enable
+		Dictionary<Type, object? []?> special_params = new Dictionary<Type, object? []?> ();
+#nullable restore
+		[Fact]
+		[AutoInitShutdown]
+		public void TestViewsDisposeCorrectly ()
+		{
+			var reference = DoTest ();
+			for (var i = 0; i < 10 && reference.IsAlive; i++) {
```

Comment:
Thank you for showing how to do this! I have been wondering if there's a way to poke into GC to see if things were really released or not.